genesis/engine/materials/FEM/elastic.py

```diff
@@ -36,8 +45,10 @@ def __init__(
 
         if model == "linear":
             self.update_stress = self.update_stress_linear
-        elif model == "stable_neohooken":
-            self.update_stress = self.update_stress_stable_neohooken
+            self.compute_energy_gradient_hessian = self.compute_energy_gradient_hessian_linear
+        elif model == "stable_neohookean":
```

Comment:
You should keep supporting the old naming and add a deprecation warning for backward compatibility. See `constraint_timeconst` for reference.